Foundation/NSData.swift

```diff
@@ -217,7 +217,11 @@ open class NSData : NSObject, NSCopying, NSMutableCopying, NSSecureCoding {
     }
     
     open var bytes: UnsafeRawPointer {
-        return UnsafeRawPointer(CFDataGetBytePtr(_cfObject))
+        guard let bytes = CFDataGetBytePtr(_cfObject) else {
+            //This could occure on empty data being encoded.
+            return UnsafeRawPointer([])
+        }
+        return UnsafeRawPointer(bytes)
```

Comment:
I would claim that a bit pattern would be more appropriate here than returning a pointer of deallocated memory: e.g. `UnsafeRawPointer(bitPattern: 0xdeadc0c0)` or something like that until we can fix the signature.